README.md

```diff
@@ -61,14 +81,77 @@ netflix.getRatingHistory(function (error, ratings) {
 })
 ```
 
+### Get Viewing History
+```javascript
+netflix.getViewingHistory(function (error, viewing) {
+  ratings === [
+    {"title":"Futurama","movieID":70153380,"yourRating":4.0, ...},
+    {"title":"Super Troopers","movieID":60022689,"yourRating":4.0, ...},
+    ...
+  ]
+})
+```
+
+### Hide Viewing History for a movie/series
+```javascript
+/**
+ * Hides viewing history for a specific movie or series
```

Comment:
Same thing about JSDoc in the README again...

Reply:
I've added a bit of text above the JSDoc, hope this is ok.